### PR TITLE
GG-25779 Do not show not needed fields on "Sign Up" dialog for on-premise Web Console

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: modules/web-console/frontend/app/modules/user/User.service.ts [hunk 1]
```diff
@@ -80,7 +81,18 @@ export default function UserFactory(
 
             sessionStorage.removeItem('demoMode');
         },
+        normalize(user: Partial<User>): Partial<User> {
```
Comment (nva) [inline modules/web-console/frontend/app/modules/user/User.service.ts]: Let's move default values to backend and remove this logic.

Comment (akuznetsov-gridgain) [reply modules/web-console/frontend/app/modules/user/User.service.ts]: Done.

Comment (Klaster1) [reply modules/web-console/frontend/app/modules/user/User.service.ts]: @akuznetsov-gridgain did you remove the frontend changes?

File: modules/web-console/e2e/testcafe/environment/envtools.js [hunk 2]
```diff
@@ -39,10 +39,14 @@ const insertTestUser = (user = testUser) => {
         .catch((err) => {throw err.message;});
 };
 
-const dropTestDB = (user) => {
+/**
+ * Removes users and their data
+ * @param user User email mask
+ */
+const dropTestDB = (user = '@example.com') => {
```
Comment (nva) [inline modules/web-console/e2e/testcafe/environment/envtools.js]:
```suggestion
const dropTestDB = (mask = '@example.com') => {
```

File: modules/web-console/e2e/testcafe/environment/envtools.js [hunk 3]
```diff
@@ -39,10 +39,14 @@ const insertTestUser = (user = testUser) => {
...
-const dropTestDB = (user) => {
+/**
+ * Removes users and their data
+ * @param user User email mask
+ */
+const dropTestDB = (mask = '@example.com') => {
     return request({
         method: 'DELETE',
-        uri: resolveUrl('/api/v1/test/users/@example.com')
+        uri: resolveUrl(`/api/v1/test/users/${user}`)
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/environment/envtools.js]:
```suggestion
        uri: resolveUrl(`/api/v1/test/users/${mask}`)
```

**Pull request merged**